src/NzbDrone.Core/DataAugmentation/Scene/SceneMappingService.cs

```diff
@@ -49,32 +48,21 @@ public class SceneMappingService : ISceneMappingService,
             _findByTvdbIdCache = cacheManager.GetCacheDictionary<List<SceneMapping>>(GetType(), "find_tvdb_id");
         }
 
-        public List<string> GetSceneNamesBySeasonNumbers(int tvdbId, IEnumerable<int> seasonNumbers)
+        public List<string> GetSceneNames(int tvdbId, List<int> seasonNumbers, List<int> sceneSeasonNumbers)
         {
-            var names = FindByTvdbId(tvdbId);
+            var mappings = FindByTvdbId(tvdbId);
 
-            if (names == null)
+            if (mappings == null)
```

Comment:
I realize this is mostly existing, but will we ever return null from `FindByTvdbId`?